src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/BatchMappingUpdateViewModels.cs

```diff
@@ -7,6 +7,36 @@
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels.Public.Data;
 
+public record LocationMappingUpdateRequest : MappingUpdateRequest
+{
+    public GeographicLevel? Level { get; init; }
+
+    public class Validator : MappingUpdateRequestValidator<LocationMappingUpdateRequest>
+    {
+        public Validator()
+        {
+            RuleFor(request => request.Level)
+                .NotNull();
+        }
+    }
+}
+
+public record FilterOptionMappingUpdateRequest : MappingUpdateRequest
+{
+    public string? FilterKey { get; init; } = null;
+
+    public class Validator : MappingUpdateRequestValidator<FilterOptionMappingUpdateRequest>
+    {
+        public Validator()
+        {
+            RuleFor(request => request.FilterKey)
+                .Cascade(CascadeMode.Stop)
+                .NotNull()
+                .NotEmpty();
+        }
+    }
+}
```

Comment:
i think this has come in a PR too early

Reply:
Yup it did somehow, whoops!  I've decided to close this PR and incorporate all changes into the PR for EES-5115 if that's OK.